pydantic_ai_slim/pydantic_ai/models/huggingface.py

```diff
@@ -299,6 +301,14 @@ async def _map_messages(
                         # please open an issue. The below code is the code to send thinking to the provider.
                         # texts.append(f'<think>\n{item.content}\n</think>')
                         pass
+                    elif isinstance(item, ServerToolCallPart):  # pragma: no cover
+                        # ServerToolCallPart represents a tool call from a remote server
+                        # This is currently never returned from huggingface
+                        pass
+                    elif isinstance(item, BuiltinToolReturnPart):  # pragma: no cover
+                        # BuiltinToolReturnPart represents a tool return from a remote server
```

Comment:
This is ambiguous as "remote server" could mean e.g. MCP. Let's make it clear this is specifically about built-in tools and drop "server" everywhere

Reply:
I think we can just drop this line FYI